src/main/java/eu/execom/pomodoroTeam/controllers/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,33 @@
+package eu.execom.pomodoroTeam.controllers;
+
+import org.jboss.logging.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import javax.persistence.EntityNotFoundException;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    private static Logger log = Logger.getLogger(GlobalExceptionHandler.class);
+
+    @ExceptionHandler(EntityNotFoundException.class)
+    public ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundException e, WebRequest request) {
+        log.info("Entity not found!");
```

Comment:
This log line does not give any information about which entity was not found. Could you extract an id from the exception, of from the request?

Reply:
Also, use `error` level to log exceptions.